WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
After putting each button in a separate div, how does this look on mobile? Screenshot, please!

I believe I asked somewhere if the floatThead will happen on mobile? I don't see any code to disable it on mobile, and I'm worried that it will take up most of the screen on mobile devices.